### PR TITLE
chore: discord.js v14 refactor

### PR DESCRIPTION
**Pull request by nakajimayoshi**
chore: discord.js v14 refactor

## Description
Refactor of the bot to be compatible with the new version of the discord.js API (v14)

-added previously unlogged changes to CHANGELOG.md
-fix embed handler
-fix command handler
-fix Client intents and partials 
-fix DM permissions

Moderation 
-fix ban.ts
-fix unban.ts 
-fix timeout.ts
-fix untimeout.ts
-fix avatar.ts 
-fix rules.ts
-fix faq.ts
-fix roleassignment.ts
-fix welcome.ts

B78XH
-new photo for irs command 

Misc
-Removed asset folder serving static image

## Discord Username
HowNowBrownCrow#2591

### CODE REVIEW

File: src/commands/moderation/ban.ts [hunk 1]
```diff
@@ -61,7 +59,7 @@ export const ban: CommandDefinition = {
     },
 };
 
-function makeSuccessfulBanEmbed(user: UserLike, reason: string): discord.MessageEmbed {
+function makeSuccessfulBanEmbed(user: UserLike, reason: string): discord.EmbedBuilder {
```
Comment (NathanInnes) [inline src/commands/moderation/ban.ts:62]:
```suggestion
function makeSuccessfulBanEmbed(user: UserLike, reason: string): EmbedBuilder {
```
Can change all ```discod.``` prefixes and just import the individual modules themselves.

File: src/commands/utils/station.ts [hunk 2]
```diff
@@ -44,9 +44,9 @@ export const station: CommandDefinition = {
                         `**City:** ${stationReport.city}`,
                         `**Latitude:** ${stationReport.latitude}°`,
                         `**Longitude:** ${stationReport.longitude}°`,
-                        `**Elevation:** ${stationReport.elevation_m} m/${stationReport.elevation_ft} ft`,,
+                        `**Elevation:** ${stationReport.elevation_m} m/${stationReport.elevation_ft} ft`, ,
```
Comment (NathanInnes) [inline src/commands/utils/station.ts:47]:
```suggestion
                        `**Elevation:** ${stationReport.elevation_m} m/${stationReport.elevation_ft} ft`, 
                        '',
```
Cleaner appearance to separate the lines.

File: src/constants.ts [hunk 3]
```diff
@@ -1,7 +1,5 @@
-// based off FlyByWire Simulations Discord Bot - https://github.com/flybywiresim/discord-bot
-
 export enum Colors {
-    HD_BLUE = '#0761e2',
```
Comment (NathanInnes) [inline src/constants.ts:4]: This one is optional however, if you wish to keep the hex format all you need to do is prefix ```0x``` before the hex. Example: ```0x0761e2```

**Review by NathanInnes — CHANGES_REQUESTED** [review]

File: src/constants.ts (hunk 3 above)
Comment (nakajimayoshi) [reply src/constants.ts:4]: Perfect, thank you!

**Pull request merged**